scripts/rust.env

```diff
@@ -12,11 +13,11 @@ COVERAGE="-C instrument-coverage"
 FUZZ_SANITIZERS="-Z sanitizer=address,leak"
 OPTIMIZE_OFF="${DEBUG_ASSERTIONS_ON} ${OVERFLOW_CHECK_ON}"
 OPTIMIZE_ON="-C opt-level=3 ${LTO} ${DEBUG_ASSERTIONS_OFF} ${OVERFLOW_CHECK_OFF}"
-OPTIMIZE_FUZZ="-C opt-level=3 ${LTO} ${DEBUG_ASSERTIONS_ON} ${OVERFLOW_CHECK_ON} ${FUZZ_SANITIZERS} ${COVERAGE}"
+OPTIMIZE_FUZZ="-C opt-level=3 ${DEBUG_ASSERTIONS_ON} ${OVERFLOW_CHECK_ON} ${FUZZ_SANITIZERS} ${COVERAGE}"
 TARGET_CPU_DEBUG="-C target-cpu=generic"
 TARGET_CPU_RELEASE="-C target-cpu=x86-64-v4"
 
-COMMON="${LINKER} ${RELRO} ${CRT} ${DEBUG}"
+COMMON="${LINKER} ${RELRO} ${DEBUG}"
 RUSTFLAGS_DEBUG="${COMMON} ${OPTIMIZE_OFF} ${TARGET_CPU_DEBUG} ${COVERAGE}"
-RUSTFLAGS_RELEASE="${COMMON} ${OPTIMIZE_ON} ${TARGET_CPU_RELEASE}"
-RUSTFLAGS_FUZZ="${COMMON} ${OPTIMIZE_FUZZ} ${TARGET_CPU_RELEASE}"
+RUSTFLAGS_RELEASE="${COMMON} ${OPTIMIZE_ON} ${TARGET_CPU_RELEASE} ${CRT_STATIC}"
+RUSTFLAGS_FUZZ="${COMMON} ${OPTIMIZE_FUZZ} ${TARGET_CPU_RELEASE} ${CRT_DYNAMIC}"
```

Comment:
Does this mean we cannot fuzz test release builds?  Does this matter?

Reply:
Looks like there are two choices here, live with this for the sanitizer or put a dynamic libc into our container.  For now, let's just live with this, and we can fix if it is an issue.